### ISSUE TITLE
Files Lockes up computer when accessing network drive

### ISSUE THREAD

**Issue by Physicstech47** [files-community/Files#13040]
### Description

I have Files on two PCs, that both have 64bit windows. One works with Files normally, and the other one locks up like a 32Bit app and requires a hard reset to free the PC. It won't even reboot or logoff without power reset.
Both of the latest Stable versions used. Is this a true 64bit app that I have installed, or is there some other issue.

### Steps To Reproduce

1. Installed thru store (paid)
2. Installed using classic (non store) version
3. One PC is 64bit AMD, the other a 4 core Intel (this is the problem PC).
4. Locks up every time I try to browse the network to any drive
5. Hard boot required. No logoff or shutdown works.

### Requirements

- e.g. Fix crash that occurs when...
Whenever the HP, Intel Processor PC launches Files and I try to open up any other file or association.
Doesn't appear to fully uninstall. The Files app icon is not full brightness, like it is disabled.

### Files Version

2.5.xx.x

### Windows Version

10.0.19045

### Log File

[debug.log](https://github.com/files-community/Files/files/12137359/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, Files doesn't support 32bit systems and a darken icon if in the start menu mean Files is updating/reinstalling.

If the problem system is not a 32bit system can you give the step to reproduce and Files version on the system that isn't working for you.

**Comment by Physicstech47** [comment]
The PC affected is as follows:

   1. HP Mini Desktop Intel 4/8 core
   2. Windows 10 Pro with updates to 22H full 64bit
   3. The install of Files works normally, but whenever I try to open or
   browse to a file on a Network Drive, the App becomes unresponsive with Task
   Manager showing 14-20% utilization.
   4. The app won't close (Files window stays open but loses focus), and
   app window shows app unresponsive.
   5. The app in windows apps & Features shows a reduced color (grayed out
   kinda).
   6. Uninstall ineffective in apps & Features, and PC will NOT shut down
   via command or logoff - Have to power off PC to kill Files App.
   7. After hard reboot, the Uninstall appears to happen successfully.
   8. Windows is up to date with all updates prior to Windows 11 upgrade.
   9. No errors listed in install .LOG file.

Whatever happens the Files App won't close, but does lose focus and other
apps work. Windows File Explorer also won't navigate the network properly,
like that service is being disabled.


   1.

**Comment by Josh65-2201** [comment]
It seem Windows is the issue since you said Windows File Explorer also has issues
Can you try running the below in an Admin CMD. When there both finished restart Windows.
`sfc /scannow`
`dism /online /cleanup-image /restorehealth`

**Comment by Physicstech47** [comment]
sfc found some corrupted files and fixed them.
dism ran and reported no issues

Will reinstall Files and let you know the result.

**Comment by Physicstech47** [comment]
Re-installed Files, this time set up colors and hidden files settings to
make sure that part worked. It appeared to work normally when accessing the
local PC.
Then connected to a network drive, and it failed immediately.
I then uninstalled, but again, It left the window open and grayed out. The
close X was available, but didn't work.
Task Manager showed Files App in the normal places, but could not kill the
app anywhere. Again, it took 13-14% resources and showed unresponsive.

I again ran sfc /scannow and it found NO corruption or problems.

I was able to connect with File Manager to all of the other drives in my
system, but not the one that was accessed by Files (apparently, still
holding the share closed)

Looks like I will not be able to use files on this PC. It is not my Video
development and Photo editing PC, so it should be ok.

Doing the hard boot now.

Thanks for your help,

Greg
Physicstech47
